README.md

```diff
@@ -958,15 +958,15 @@ in inheritance.
       :delta => 4,
       :alpha => 1,
```

Comment:
You need to update all of the other examples (like this one) that use hashes to use the 1.9 syntax.